boards/xtensa/intel_adsp_cavs25_2core/Kconfig.defconfig

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_INTEL_ADSP_CAVS25
+
+config BOARD
+	default "intel_adsp_cavs25_2core"
+
+config CAVS_ICTL_0_OFFSET
+	default 6
+config CAVS_ICTL_1_OFFSET
+	default 10
+config CAVS_ICTL_2_OFFSET
+	default 13
+config CAVS_ICTL_3_OFFSET
+	default 16
```

Comment:
Don't think these values change, do they?  Probably best to leave them defined at the SOC level somehow than cut/paste.

Reply:
Let me check. I'll at least refer to the base cavs25 definitions. Agreed these should not be copy'n'pasted here.